testing_and_setup/compass/ocean/global_ocean/scripts/create_E3SM_coupling_files.ini

```diff
@@ -0,0 +1,27 @@
+[main]
+mesh_name = oQU240
```

Comment:
Another to-do: To be general, COMPASS needs to read its own resolution name. There may be a resolution variable available within COMPASS (I have not seen it). Otherwise, `create_E3SM_coupling_files.py` could try to parse its path to automatically collect the resolution string `QU240`.

Reply:
> There may be a resolution variable available within COMPASS (I have not seen it).

An argument gets added to the config file here:
https://github.com/MPAS-Dev/MPAS-Model/blob/master/testing_and_setup/compass/setup_testcase.py#L1640
```
        config.set('script_input_arguments', 'resolution', args.resolution)
```

However, we are still using the python 2 version of the config parser:
https://github.com/MPAS-Dev/MPAS-Model/blob/master/testing_and_setup/compass/setup_testcase.py#L25
available in python 3 via the `six` package.
```
from six.moves import configparser
```
That config parser doesn't let you refer to config options in sections other than the one you're in.  If we switch to the python 3 config parser exclusively (dropping python 2 support), we could use "extended interpolation":
https://docs.python.org/3/library/configparser.html#configparser.ExtendedInterpolation
do something like:
```
[main]
mesh_name = o${script_input_arguments:resolution}
```

Reply:
I think this would be a pretty trivial change as long as we're willing to drop python 2 support in COMPASS.

Reply:
I take this back.  This is a separate config file, parsed by a separate script so there isn't a way to get resolution from the config object in `setup_testcase.py`.  It is probably necessary to parse it out of the path.